apps/e2e/helpers/cr-sqlite.ts

```diff
@@ -1,55 +1,69 @@
 import type { DB } from "@vlcn.io/crsqlite-wasm";
-import {
-	Customer,
-	PlacedSupplierOrder,
-	PlacedSupplierOrderLine,
-	Supplier,
-	PossibleSupplierOrderLine,
-	ReconciliationOrderLine,
-	ReconciliationOrder,
-	DBCustomerOrderLine
-} from "./types";
+
+import { Customer, PlacedSupplierOrder, PlacedSupplierOrderLine, Supplier, CustomerOrderLine, SupplierOrderLine } from "./types";
 import { BookData } from "@librocco/shared";
 
 // #region books
+import { Page } from "@playwright/test";
+
+import type {} from "./types";
+
+// #region books
+
+// Extend the window object with the db
+declare global {
+	interface Window {
+		_db: DB;
+		db_ready: boolean;
+
+		books: Record<string, any>;
```

Comment:
I think it's good to resist the temptation to better specify this `any` here. I second not having types for the return objects of these functions.